src/passkey_child/passkey_child.c

```diff
@@ -84,6 +84,15 @@ int main(int argc, const char *argv[])
             ERROR("Error getting assertion data.\n");
             goto done;
         }
+    } else if (data.action == ACTION_VERIFY_ASSERT) {
+        ret = verify_assert_data(&data);
+        if (ret == EOK) {
+            printf("Verification success.\n");
```

Comment:
You should use `PRINT` instead of `printf` all around the code to ensure it gets translated.

Reply:
Added a new commit to replace them.